.github/workflows/docs-lint.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Lint Docs
```

Comment:
First time using GitHub Actions, so special attention is needed here.

Reply:
This looks good to me! Nicely done :)